scripts/service-worker-integration.ts

```diff
@@ -0,0 +1,64 @@
+import type { APIRoute, AstroIntegration } from 'astro';
```

Comment:
I think I would move this file to `config/astro/service-worker-integration.ts` or `config/astro/integrations/service-worker.ts`, since it's basically a configuration snippet that's loaded into the main `astro.config.ts`. Whereas `/scripts/` contains all scripts that are executed from a cli context.